src/UserSpaceInstrumentation/TestLib.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "TestLib.h"
+
+#include <cstdio>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+
+namespace orbit_user_space_instrumentation {
+
+namespace {
+
+std::filesystem::path GetTmpFilePath() {
+  static std::filesystem::path p = std::tmpnam(nullptr);
+  return p;
+}
+
+}  // namespace
+
+void InitTestLib() {
+  auto p = GetTmpFilePath();
+  std::cout << "Init Lib. Tmp file is: " << p << std::endl;
+}
+
+void UseTestLib(const std::string& s) {
+  auto p = GetTmpFilePath();
+  std::ofstream ofs;
```

Comment:
I get that this is just for testing, but still I thought we wanted to get rid of `std` streams because of exceptions? I guess we should be consistent. There are utilities in `OrbitBase` to read and write an entire text file (after all they are nicer to use than stream directly).

Reply:
I'd like to keep this as "vanilla" as possible for so I decided against using OrbitBase. 

This will not be linked to Orbit or even any test so I think exceptions don't matter in this case.